priv/static/phoenix_live_view.js

```diff
@@ -4329,6 +4329,7 @@ removing illegal node: "${(childNode.outerHTML || childNode.nodeValue).trim()}"
         });
       }
       if (liveview_version !== this.liveSocket.version()) {
+        // See https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#static_changed?/1 for tips to address this
```

Comment:
```suggestion
        // Either the user has a mismatched version - for example using LiveView from npm instead of
        // the bundled version from the hex package - or, more likely, the assets are stale because of a
        // deployment. In that case, see
        // https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#static_changed?/1
        // for tips to address this.
```